add file connector example (#1118)
wso2/docs-ei: en/micro-integrator/docs/references/connectors/configure-connector-exporter-project.md

## Configure the connector in WSO2 Integration Studio

1. Open WSO2 Integration Studio and create an ESB Solution Project.

> <p><img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" /></p>

Reviewer: Let's add images in line with the numbered list. Otherwise, this looks disjointed and out of sync with the list numbers.

Author: This is done.

> 2. Right click on the project that you created and click on 'Add or Remove Connector' -> Add Connector.

Reviewer: ```suggestion
2. Right click on the project that you created and click on **Add or Remove Connector** -> **Add Connector**.
```

Let's bold all terms that come directly from the UI.

Author: Done

<p><img src="../../assets/img/connectors/add-connector.png" title="Add or Remove Connector" width="800" alt="Add or Remove Connector" /></p>

> 3. It will be navigated to the WSO2 Connector Store. 

Reviewer: ```suggestion
3. You are directed to the WSO2 Connector Store. 
```

Author: Done.

4. Search for File Connector and download it to the workspace. 
<p><img src="../../assets/img/connectors/search-connector.png" title="Search Connector in the Connector Store" width="800" alt="Search Connector in the Connector Store" /></p>

> 5. Now it needs to add a Connector Exporter Project. 

Reviewer: ```suggestion
5. Now you need to add a Connector Exporter Project. 
```

Merge this point with the next one. All items in the list must have actions.

Author: This line was removed.

6. Go to File -> New -> Other -> WSO2 -> Extensions -> Project Types -> Connector Exporter Project
<p><img src="../../assets/img/connectors/connector-exporter-project-1.png" title="Add Connector Exporter Project" width="800" alt="Add Connector Exporter Project" /></p>

> <p><img src="../../assets/img/connectors/connector-exporter-project-2.png" title="Add Connector Exporter Project" width="800" alt="Add Connector Exporter Project" /></p>

Reviewer: Shall we create a separate list item for this or have some explanation before we add this? It's not good for readability to have two images one after the other without text in the middle.

Author: Done.

> 7. Provide a name for Connector Exporter Project and in the next screen tick on 'Specify the parent from workspace' and select the File Connector project. 

Reviewer: ```suggestion
7. Enter a name for the Connector Exporter Project and in the next screen select **Specify the parent from workspace** and select the File Connector project. 
```

I feel all the list items in this page seem to be too compressed. It's like we are trying to cover multiple steps in one step itself. Shall we rethink this and have one step per list item?

Author: Splitted them to two steps.

<p><img src="../../assets/img/connectors/connector-exporter-project-naming.png" title="Naming Connector Exporter Project" width="800" alt="Naming Connector Exporter Project" /></p>

> 8. 10. Now it needs to add the Connector to Connector Exporter project that we just created. Right Click on the Connector Exporter project -> New -> Add Remove Connectors -> Add Connector -> Add from Workspace -> File Connector

Reviewer: ```suggestion
8. Now you need to add the Connector to Connector Exporter project that you just created. Right click on the Connector Exporter project and select **New** -> **Add Remove Connectors** -> **Add Connector** -> **Add from Workspace** -> **File Connector**
```

There are too many images under this. Also the numbering is wrong when you render this as it displays as a sub list. Also let's have only one screenshot here. This is too much and affects readability.

Author: This is added as a separate page which is 'Exporting artifacts' under the subsection of  'Creating Connector Exporter Project'.

> 8. 10. Now it needs to add the Connector to Connector Exporter project that we just created. Right Click on the Connector Exporter project -> New -> Add Remove Connectors -> Add Connector -> Add from Workspace -> File Connector

Reviewer 2: We need to write it in a way not specific to the `File connector` as this is a common doc.

Author: This is done.
